packages/types/src/localization.ts

```diff
@@ -631,11 +631,25 @@ type _LocalizationResource = {
         automaticSuggestionOption__label: LocalizationValue;
         automaticSuggestionOption__description: LocalizationValue;
         formButton__save: LocalizationValue;
+        calloutLabels: {
+          prefix_single: LocalizationValue;
+          prefix_many: LocalizationValue;
+          pendingInvitationCount_single: LocalizationValue;
```

Comment:
Just curious, are we using this pattern anywhere else?

Reply:
We have [this](https://github.com/clerkinc/javascript/blob/1663aaae704b8a0f62e31acab440584e681666b3/packages/localizations/src/en-US.ts#L535) for plurals

and [this](https://github.com/clerkinc/javascript/blob/1663aaae704b8a0f62e31acab440584e681666b3/packages/localizations/src/en-US.ts#L722C13-L722C13) that uses some kind of concatenation but not both